client/my-sites/customer-home/cards/actions/quick-links/style.scss

```diff
@@ -102,6 +107,11 @@
 		}
 	}
 
+	&:focus-visible {
+		box-shadow: inset 0 0 0 2px var( --color-primary );
```

Comment:
This works  in chrome, I can't seem to tab to the quick link elements at all in Firefox on macos. (I don't think that's related to this PR though)

Reply:
I agree, probably worth creating an issue to track all FF problems!